docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -2,9 +2,14 @@
 [[ref-daemon-config-files-provisiond]]
 = Provisiond
 
-The provision daemon handles scanning of requisition data to xref:operation:provisioning/introduction.adoc[provision] into monitored nodes.
+The provision daemon scans nodes from requisition and import files, and from newSuspect events.
+The scanned nodes are then added to the database.
+Note that only nodes in the database are monitored by other daemons.
 
-It is recommended that you update configuration settings using the {page-component-title} UI.
+NewSuspect events are triggered by Discover scans, and by trapd and syslogd if those daemons are configured to create the event.
+When configured properly, trapd and syslogd create a newSuspect event if an incoming message does not match any existing nodes.
```

Comment:
This line is redundant to the previous sentence.

Reply:
```suggestion
```